### DISCUSSION TITLE
GLSL renderer issues

### DISCUSSION THREAD

**Discussion by Correct-Syntax** [GimelStudio/GimelStudio#178] [General]
@yonMaor @iwoithe Let's continue the discussion from #147 here. 😄

**Comment by Correct-Syntax** [comment]
@yonMaor does the halftone shader also work when you manually set the size value?

**Comment by yonMaor** [comment]
Good idea, but no, the halftone node still has other issues that I haven't been able to solve yet.
I'll work on it a little and see if I can make it work with the resolution hard coded (like I did for the noise image node)

**Reply by Correct-Syntax** [reply]
Ah, okay. I was trying to pinpoint whether the size was consistently the issue. But I didn't think so since the box blur shader I worked on didn't have an issues.

**Reply by yonMaor** [reply]
It's not the size itself, but I do believe that there are still issues with the parameters that are passed into the fragment shader from the vertex shader
